src/Symfony/Component/Routing/Loader/Configurator/RoutingConfigurator.php

```diff
@@ -36,7 +36,7 @@ public function __construct(RouteCollection $collection, PhpFileLoader $loader,
     /**
      * @return ImportConfigurator
      */
-    final public function import($resource, $type = null, $ignoreErrors = false)
+    final public function import($resource, $type = null, int $ignoreErrors = PhpFileLoader::IGNORE_NONE)
```

Comment:
Changing this type already will cause problems when passing a bool, it should be changed in Symfony 5.0 instead.

Reply:
I have removed the type to have less BC breaks.